src/lsi/utils/ansible_lsi.py

```diff
@@ -0,0 +1,129 @@
+# Copyright (c) 2015, Narrative Science
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright notice, this
+#   list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+
+import ansible.playbook
+import ansible.runner
+import ansible.inventory
+from ansible import callbacks
+from ansible import utils
+import json
+import os
+
+utils.VERBOSITY = 0
+stats = callbacks.AggregateStats()
+playbook_cb = callbacks.PlaybookCallbacks(verbose=utils.VERBOSITY)
+runner_cb = callbacks.PlaybookRunnerCallbacks(stats, verbose=utils.VERBOSITY)
+
+def build_inventory(inventory_dict, query_group):
+    '''
+    Builds dynamic inventory list for use with ansible commands
+
+    :param inventory_dict: The dictionary to create inventory
+    :type inventory_dict: ``dict``
+    :param query_group: Subset group created from lsi filters
+    :type query_group: ``str``
+    
+    :return: Inventory file name
+    :rtype: ``str``
+    '''
+    _INVENTORY_FILE = os.path.expanduser('~/.lsi-ansible-inventory')
+    inventory_file = _INVENTORY_FILE
```

Comment:
You don't need the all caps variable.  All caps implies it's a module level constant.